modules/teacher.rb

```diff
@@ -1,11 +1,11 @@
-require '../library'
+require_relative '../person'
 class Teacher < Person
-  def initialize(name, age, parent_permission, specialization)
-    super(name, age, parent_permission)
+  def initialize(age, specialization, name = 'unknown')
```

Comment:
- [ ] There are some errors in your app this in your app as you can see in the screenshot below ⬇️  Please be sure to fix them in order to make your app able to create a teacher and also a rental to make it works properly  👍 

![Capture](https://user-images.githubusercontent.com/6615394/225905357-0d001f23-35e8-49c7-8939-c0bb8d9da145.PNG)